crates/aiken-lsp/src/server/lsp_project.rs

```diff
@@ -61,6 +59,8 @@ impl LspProject {
             self.modules.insert(module.name.to_string(), module);
         }
 
+        result?;
```

Comment:
ah nice

Reply:
It puzzled me for a while this one. Debugging the lsp server isn't quite easy and it took me some time to figure out why the server wasn't aware of any module.